### DISCUSSION TITLE
Access Manager for USB ports on the machine

### DISCUSSION THREAD

**Discussion by sylwester-majcher** [lithnet/access-manager#71] [Ideas] [answered]
Hi, we will be testing last release of Access Manager. Thanks a lot for this product it looks amazing.
From my end I am really interesting of JIT feature!
If we mean about new idea and because this is Access Manager...why not add some feature to manage USB access for removable drive for example? I dont thint about managing guides of devices. Some of simple solution to turn on or turn off usb for removeble drives. I know there is GPO entries we can use but when computers are on the vpn and some times you need to change computer groups assosiation, this is nightmare.
So simply my idea: in freeware license: poor turn on/off management for usb remov drives
In enterprise: more granular as managing of guide or model or producer etc

**Comment by ryannewington** [comment, marked as answer]
@sylwester-majcher 

Thanks for the feedback and the USB ports idea. I don't think we'll be able to implement this as it currently stands, as it would require architecture changes to the product that would be fairly extensive. However, we'll keep it in mind for future versions of the product if we can find a fit for this.
